### PR TITLE
chore(main): release 0.2.0

### PR DESCRIPTION
**Pull request by tpnam0901**
:robot: I have created a release *beep* *boop*
---

### Features

* add a copy of google colab ([d396dd3](https://github.com/tpnam0901/QTSeg/commit/d396dd3645ac9b18ac6925949e8b001ec3149bab))
* add base config ([4f779c6](https://github.com/tpnam0901/QTSeg/commit/4f779c672448fd20816a0653cef72a7903710c6d))
* add BKAI, BUSI, ISIC configs ([ec3ca3e](https://github.com/tpnam0901/QTSeg/commit/ec3ca3e30dbb5eb5c467dea0a3719b8816d7f7ae))
* add dataset, dataloader for BKAI, BUSI, ISIC ([829381a](https://github.com/tpnam0901/QTSeg/commit/829381ab0ebe03de0b15b05bf6a0e844b83a6985))
* add environment settings ([4ec3721](https://github.com/tpnam0901/QTSeg/commit/4ec3721010b8fae6fed94b3f0a7a126dc099b52d))
* add FPN Encoder module ([5ce738f](https://github.com/tpnam0901/QTSeg/commit/5ce738f8471e794d0e62c6029e7658bf275b8625))
* add loss function, metrics, lr schedulers ([607819c](https://github.com/tpnam0901/QTSeg/commit/607819cbfa1db3154275ad27408d4823ef54792b))
* add MQM Decoder module ([6fed401](https://github.com/tpnam0901/QTSeg/commit/6fed401b72f102f83fcd00e870f0e47acaf2fdb5))
* add optimizers custom ([406ba6e](https://github.com/tpnam0901/QTSeg/commit/406ba6e153724f8989cf56ff04f036ef1b3c4195))
* add preprocess tools for BKAI, BUSI ([823d4c7](https://github.com/tpnam0901/QTSeg/commit/823d4c727971d5be64b3fbe8759d9a10af60a41b))
* add QTSeg model ([7f8657d](https://github.com/tpnam0901/QTSeg/commit/7f8657d66341823100c51f57e69f2eed37d44330))
* add train, eval, infer scripts ([87e4304](https://github.com/tpnam0901/QTSeg/commit/87e430448051f4150ebb08ce1064618638505472))
* init repo ([0184d7c](https://github.com/tpnam0901/QTSeg/commit/0184d7cfd7c4fb97fcae9f151fb3b50c6a83d320))

### Bug Fixes

* BUSI, ISIC dataloader config ([17fdb96](https://github.com/tpnam0901/QTSeg/commit/17fdb965adcaa16ce7e843d8f8cfe1c45baea16a))
* wrong epoch increment ([cee022b](https://github.com/tpnam0901/QTSeg/commit/cee022b4578692b96b30e5f10e195e950e786506))

### Documentation

* add arxiv, citation ([7b663e8](https://github.com/tpnam0901/QTSeg/commit/7b663e8809eebee426a76ff34d253f0621a23248))
* add colab link ([fe69f84](https://github.com/tpnam0901/QTSeg/commit/fe69f8412478cd347034abd2cec3073860cc5785))
* add readme and preview image ([81ba765](https://github.com/tpnam0901/QTSeg/commit/81ba765295c9193c16a7c44c12fd7c838a48312d))
* fix BKAI, BUSI link ([ecf90c6](https://github.com/tpnam0901/QTSeg/commit/ecf90c656afca528b22be2d8747b5180b3c4f38c))
* fix title ([ea5c531](https://github.com/tpnam0901/QTSeg/commit/ea5c531d13ad06228311ef9f560be64aaa412ca3))
* rearrange figure position ([2e8448f](https://github.com/tpnam0901/QTSeg/commit/2e8448f2d1a2d18c84cbe38bdde4abd104f3dfeb))
* update abstract ([15b5131](https://github.com/tpnam0901/QTSeg/commit/15b51312c5ab6dad64bd2a36fa5d9701c615b8eb))

---
This PR was generated with [Release Please](https://github.com/googleapis/release-please). See [documentation](https://github.com/googleapis/release-please#release-please).

### CODE REVIEW

**Comment by tpnam0901** [comment]
:robot: Release is at https://github.com/tpnam0901/QTSeg/releases/tag/v0.2.0 :sunflower:
